### ISSUE TITLE
cordova.plugins.Keyboard.isVisivle   always  return false ,why?

### ISSUE THREAD

**Issue by 519818538** [driftyco/ionic-plugin-keyboard#186]

**Comment by archangellcc** [comment]
Just in $ionicPlatform.registerBackButtonAction() function
$cordovaKeyboard.isVisible() is always return false.

**Comment by 519818538** [comment]
Yeah.But i  want to close the keyboard after pressing the back button for the first time.

**Comment by archangellcc** [comment]
I have the same problem with you, also need the solution. I post this question in stackoverflow , ionic WorldWide ... not enough people to pay attention.  
ps:
I found this problem arise in android 4.4 and be solved in android 5.0.
But, I still want to know how to avoid that in andorid 4.4.

**Comment by archangellcc** [comment]
Hey，I find a solution about that question.

                window.addEventListener('native.keyboardhide', keyboardHideHandler);
                function keyboardHideHandler(e){
                  var test=$ionicPlatform.registerBackButtonAction(function (e) {
                  },1000);
                    $timeout(function() {
                      test();
                    }, 100);
                }
I know this method is not best practice, but it can work. If you find more better method or this method production some bugs, please let me know.
qq 342897204

**Comment by 519818538** [comment]
Ok,thanks for your sharing.
